app/assets/javascripts/admin/addon/components/webhook-events.hbs

```diff
@@ -14,16 +15,34 @@
     <LoadMore @selector=".web-hook-events li" @action={{this.loadMore}}>
       <div class="web-hook-events content-list">
         <div class="heading-container">
-          <div class="col heading first status">{{i18n "admin.web_hooks.events.status"}}</div>
-          <div class="col heading event-id">{{i18n "admin.web_hooks.events.event_id"}}</div>
-          <div class="col heading timestamp">{{i18n "admin.web_hooks.events.timestamp"}}</div>
-          <div class="col heading completion">{{i18n "admin.web_hooks.events.completion"}}</div>
-          <div class="col heading actions">{{i18n "admin.web_hooks.events.actions"}}</div>
+          <div class="col heading first status">{{i18n
```

Comment:
I believe these lines could have been formatted in another more beautiful way, but I understand that it would slightly change the semantics.

Reply:
Yeah, it's unfortunate… it sometimes makes the template a bit unreadable:

```hbs
<span class="d-button-label">{{html-safe this.computedLabel}}{{#if
    this.ellipsis
  }}&hellip;{{/if}}</span>
```

```hbs
{{#if tag.pmOnly}}{{d-icon "envelope"}}{{/if}}{{#if tag.totalCount}}
  <span class="tag-count">x {{tag.totalCount}}</span>{{/if}}
```

```hbs
<tr><td colspan="4">{{i18n
      "admin.email.incoming_emails.none"
    }}</td></tr>
```

"prettier" it ain't 👀

Reply:
Yeah agreed - in cases like these we'll need to make some manual tweaks (i.e. change the semantics) to make things look nicer. e.g. in the lines @nbianca mentioned we can do this. Technically it's changing the whitespace in the HTML, so prettier isn't willing to do it automatically:

```handlebars
<div class="heading-container">
  <div class="col heading first status">
    {{i18n "admin.web_hooks.events.status"}}
  </div>
  <div class="col heading event-id">
    {{i18n "admin.web_hooks.events.event_id"}}
  </div>
  <div class="col heading timestamp">
    {{i18n "admin.web_hooks.events.timestamp"}}
  </div>
  <div class="col heading completion">
    {{i18n "admin.web_hooks.events.completion"}}
  </div>
  <div class="col heading actions">
    {{i18n "admin.web_hooks.events.actions"}}
  </div>
</div>
```

I think I'd prefer to get a '100% safe' automated commit merged first, and then we can fix up individual cases like this as/when we find them.

Reply:
I skimmed through the diff and added a commit which fixes up the most obviously bad cases, including the ones you mentioned @cvx @nbianca